perfkitbenchmarker/providers/openstack/flags.py

```diff
@@ -44,10 +44,28 @@
                     'defaults to $NOVA_ENDPOINT_TYPE.')
 
 flags.DEFINE_string('openstack_public_network', None,
-                    'Name of OpenStack public network')
+                    '(DEPRECATED: Use openstack_floating_ip_pool) '
+                    'Name of OpenStack public network.')
 
 flags.DEFINE_string('openstack_private_network', 'private',
-                    'Name of OpenStack private network')
+                    '(DEPRECATED: Use openstack_network) '
+                    'Name of OpenStack private network.')
+
+flags.DEFINE_string('openstack_network', 'private',
+                    'Name of OpenStack network. Network provides '
+                    'automatically allocated fixed-IP addresses to attached '
+                    'instances. Typically, network is used for internal '
```

Comment:
Unless network is a variable, I think descriptions like this would read easier as "the network", "this network", or similar.